Code/GraphMol/MolAlign/Wrap/rdMolAlign.cpp

```diff
@@ -661,6 +695,59 @@ BOOST_PYTHON_MODULE(rdMolAlign) {
       docString.c_str());
 
   docString =
+      "Returns the RMS between two conformations.\n\
+     \n\
+     By default, the conformers will be aligned to the first conformer\n\
+     of the molecule (i.e. the reference) before RMS calculation and,\n\
+     as a side-effect, will be left in the aligned state.\n\
+     \n\
+     ARGUMENTS:\
+      - mol         the molecule\n\
+      - confId1     the id of the first conformer\n\
+      - confId2     the id of the second conformer\n\
+      - atomIds     (optional) list of atom ids to use a points for\n\
+                    alingment - defaults to all atoms\n\
+      - prealigned  (optional) by default the conformers are assumed\n\
+                    be unaligned and will therefore be aligned to the\n\
+                    first conformer\n\
+     \n\
+     RETURNS\n\
+     RMSD value\n\
+     \n";
+  python::def("GetConformerRMS", RDKit::getConformerRMS,
```

Comment:
no, it's all over the place. Normally when you are writing RDKit code on the C++ side you are inside a `namespace RDKit` block, so you don't notice. It shows up in the C++ tests a lot though.